lib/appmetrics-prometheus.js

```diff
@@ -273,6 +262,267 @@ function stringHttpRequestDuration(httpURLDataList) {
   return content;
 }
 
+function stringProcessUptime() {
+  return [
+    '# HELP process_uptime_count_seconds The number of seconds for which the current Node.js process has been running',
+    '# TYPE process_uptime_count_seconds counter',
+    `process_uptime_count_seconds ${process.uptime()}\n`,
+  ];
+}
+
+function stringLoopData(latestLoopEvent) {
```

Comment:
the `event_loop_` should be `nodejs_eventloop_` -- "event" doesn't identify the runtime.

promclient uses names that use RUNTIME_THING_, as does go:
- https://github.com/siimon/prom-client/blob/cc9ce780ce59207c48fec79c539de947801348ee/lib/metrics/eventLoopLag.js#L5
- https://github.com/prometheus/client_golang/blob/333f01cef0d61f9ef05ada3d94e00e69c8d5cdda/prometheus/go_collector.go#L66-L81